app/frontend/components/pages/login/loginPage.tsx

```diff
@@ -214,37 +212,39 @@ const LoginPage = () => {
     walletLoadingError,
     shouldShowGenerateMnemonicDialog,
     shouldShowWalletLoadingErrorModal,
-    shouldShowStakingBanner,
     autoLogin,
-    errorBannerContent,
   } = useSelector((state: State) => ({
     authMethod: state.authMethod,
     shouldShowLogoutNotification: state.shouldShowLogoutNotification,
     walletLoadingError: state.walletLoadingError,
     shouldShowGenerateMnemonicDialog: state.shouldShowGenerateMnemonicDialog,
     shouldShowWalletLoadingErrorModal: state.shouldShowWalletLoadingErrorModal,
-    shouldShowStakingBanner: state.shouldShowStakingBanner,
     autoLogin: state.autoLogin,
-    errorBannerContent: state.errorBannerContent,
   }))
-  const {
-    closeStakingBanner,
-    setAuthMethod,
-    closeWalletLoadingErrorModal,
-    loadErrorBannerContent,
-  } = useActions(actions)
+  const {setAuthMethod, closeWalletLoadingErrorModal} = useActions(actions)
 
   useEffect(() => {
     if (autoLogin && authMethod !== AuthMethodType.MNEMONIC) {
       setAuthMethod(AuthMethodType.MNEMONIC)
     }
-    loadErrorBannerContent()
   }, []) // eslint-disable-line
 
   return (
     <div className="page-wrapper">
-      {shouldShowStakingBanner && <StakingBanner onRequestClose={() => closeStakingBanner()} />}
-      {errorBannerContent && <ErrorBanner message={errorBannerContent} />}
+      <div className="banner success">
```

Comment:
this should be temporary, after couple of week we will revert to previous behaviour